src/ocl/Makefile

```diff
@@ -80,8 +80,14 @@ endif
 # !!! End of control section. Everything below is not designed to be modified by user
 #=======================================================================================================================
 
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+	LDLIBS  += -framework OpenCL
```

Comment:
Please make two-spaces indent. The same two lines below. 

The rest is fine - ready for merge.

Reply:
Done. I didn't pay attention to the auto-indent.